ngrinder-controller/src/main/java/org/ngrinder/common/constant/ControllerConstants.java

```diff
@@ -82,4 +82,7 @@ public interface ControllerConstants {
 	String PROP_CONTROLLER_ENABLE_STATISTICS = "controller.enable_statistics";
 	String PROP_CONTROLLER_CSV_SEPARATOR = "controller.csv_separator";
 	String PROP_CONTROLLER_GITHUB_BASE_URL = "controller.github_base_url";
+	String PROP_CONTROLLER_IGNORE_TOO_LOW_TPS = "controller.ignore.too_low_tps";
+	String PROP_CONTROLLER_IGNORE_TOO_MANY_ERRORS = "controller.ignore.too_many_errors";
+	String PROP_CONTROLLER_IGNORE_TOO_MUCH_TRAFFIC_ON_REGION = "controller.ignore.too_much_traffic_on_region";
```

Comment:
controller.ignore_too_low_tps
controller.ignore_too_many_errors
controller.ignore_too_much_traffic_on_region

might be better.

Reply:
agree :)